### DISCUSSION TITLE
custom 3d model doesn't seem to be converting correctly

### DISCUSSION THREAD

**Discussion by Kylorde** [Kas-tle/java2bedrock.sh#20121] [Converter Help]
I've tried changing the settings a bit but the result seems the same regardless. the item in the inventory isn't displaying anything but the host items png and the 3d item is shifted in the players hand and isn't displaying pixels that aren't opaque. I'm open to any suggestions but am very ignorant on working with bedrock.

my goal was just to get a set of items from my first datapack to be seen on both ends (first time working with geyser)

![image_2024-02-10_151948337](https://github.com/Kas-tle/java2bedrock.sh/assets/47133142/ecc55bf8-6c51-4b4e-b71d-9bee654cce49) 
this is on bedrock

![image_2024-02-10_152337895](https://github.com/Kas-tle/java2bedrock.sh/assets/47133142/46fd8f09-0abe-4fd7-a21a-815fed9388ce)
this is how its supposed to look, on java

**Comment by Kylorde** [comment]
I made a new texture pack that is opaque as a temporary solution but now no matter that i do, settings i change, redoing the texturepack, i just get exit code 1 and i can't get a new bedrock texturepack.

it says there isn't a pack.mcmeta when there clearly is. I've tried changing pack versions but that doesn't seem to help

**Reply by HotsukokuAito** [reply]
Can u send here the link of ur file?
